Multisig/UI/App/UIKit/UI Library/UIView+UINib.swift

```diff
@@ -8,7 +8,7 @@
 
 import UIKit
 
-extension UITableViewCell {
+extension UIView {
```

Comment:
I think the UIView is the wrong place for the `reuseID`. I think, this variable should be present only in reusable classes - UITableViewCell, UITableViewHeaderFooterView, and UICollectionViewCell, (and possibly collection view headers). 

Therefore, it seems that the better way to handle reuse of this code is to declare a protocol with default implementation and make those classes adopt this protocol.